src/ucs/sys/sock.c

```diff
@@ -37,6 +37,15 @@ typedef ssize_t (*ucs_socket_iov_func_t)(int fd, const struct msghdr *msg,
                                          int flags);
 
 
+static void ucs_socket_print_error_info(int sys_errno)
+{
+    if (sys_errno == EMFILE) {
+        ucs_error("the maximal number of files that could be opened "
+                  "simultaneously (%d) was reached, try to increase the limit",
```

Comment:
can you mention `ulimit -n` please? to tell the user how to increase the limit

Reply:
good catch, done